draft-ietf-add-resolver-info.md

```diff
@@ -210,9 +210,9 @@ resolver.example.net. 7200 IN RESINFO qnamemin exterr=15,16,17
    If a resolver supports DDR but does not support RESINFO, the client	
 	can receive a positive RESINFO response from an upstream DNS resolver	
 	or an attacker.  To prevent such an attack, resolvers supporting DDR	
-	MUST convey the "sig" attribute. In addition to the validaton checks in {{Section 4.2 of I-D.ietf-add-ddr}},  DNS clients using DDR for encrypted	
+	MUST convey the "sig" attribute. DNS clients using DDR for encrypted	
 	resolver discovery querying for a RESINFO RR MUST validate the	
-	signature in the "sig" attribute for data origin authentication.  If	
+	signature in the "sig" attribute for data origin authentication, including ensuring that the certificate that is used for the signature also contains the IP address that DDR uses for validation as per {{Section 4.2 of I-D.ietf-add-ddr}}.  If	
 	the signature validation fails, the DNS client MUST reject the	
```

Comment:
I propose to modify the line as follows:
DNS clients using DDR for encrypted resolver discovery querying for a RESINFO RR MUST validate the signature in the "sig" attribute for data origin authentication using the public key in the certificate, including ensuring that the certificate contains the IP address that DDR uses for validation as per {{Section 4.2 of I-D.ietf-add-ddr}}.

Reply:
@tireddy2: OK. Please see [0813368](https://github.com/boucadair/add-resolver-information/pull/9/commits/08133683b6f9a2a12f8c57e19dd867a82daa2c44)